clang/test/SemaCXX/attr-suppress.cpp

```diff
@@ -0,0 +1,62 @@
+// RUN: %clang_cc1 -std=c++11 -fsyntax-only %s -verify
+
+[[gsl::suppress("globally")]];
+
+namespace N {
+[[gsl::suppress("in-a-namespace")]];
+}
+
+[[gsl::suppress("readability-identifier-naming")]] void f_() {
+  int *p;
+  [[gsl::suppress("type", "bounds")]] {
+    p = reinterpret_cast<int *>(7);
+  }
+
+  [[gsl::suppress]] int x;       // expected-error {{'suppress' attribute takes at least 1 argument}}
+  [[gsl::suppress()]] int y;     // expected-error {{'suppress' attribute takes at least 1 argument}}
+  int [[gsl::suppress("r")]] z;  // expected-error {{'suppress' attribute cannot be applied to types}}
+  [[gsl::suppress(f_)]] float f; // expected-error {{expected string literal as argument of 'suppress' attribute}}
```

Comment:
This seems to be incorrect for `stable/20230725` because the wording only seem implemented in https://reviews.llvm.org/D156237, which was only merged in August 15th.

The generated message in this branch is ”'suppress' attribute requires a string”.

All three instances of ”expected string literal…” in this change should be changed to the former spelling.

Reply:
Uh-oh! I'll fix.

Reply:
Looks like I forgot to run tests entirely. Sorry!